alembic/versions/85f409b2eeeb_add_ticket_number.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: 85f409b2eeeb
+Revises: dd72c72e23a2
+Create Date: 2022-11-11 11:27:17.102596
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "85f409b2eeeb"
+down_revision = "dd72c72e23a2"
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column("analysis", sa.Column("ticket_id", sa.String(32), nullable=True))
```

Comment:
String or int that is the question....

Also, here you are using `ticket_id`. It is great to standardize this in both apps since we are migrating the data in between. It will be less confusing in 3 months when someone else reads the code